Core/GraphQL/GraphQLFiles/sell_fragment_order.graphql

```diff
@@ -0,0 +1,94 @@
+fragment FragmentOrder on Order {
+  id
+  orderNumber
+  qrCodeUrl
+  collectionDate
+  status
+  state {
+    workingState
+    status
+    translations {
+      language
+      title
+      desciption
+    }
+  }
+  paymentStatus
+  grossAmount
+  discountAmount
+  netAmount
+  estimatedAt
+  createdAt
+  updatedAt
+  items {
+    id
+    product{...FragmentProduct}
+    productVariant {
```

Comment:
I think we can create a `productVariant` fragment so it can be reused here as well as `product.variants`